hr_attendance_management/models/hr_employee.py

```diff
@@ -42,12 +42,24 @@ class HrEmployee(models.Model):
 
     today_hour_formatted = fields.Char(compute='_compute_today_hour_formatted')
 
-    work_location = fields.Many2one('hr.attendance.location')
+    work_location_id = fields.Many2one('hr.attendance.location',
+                                       string='Work Location')
+
+    work_location = fields.Char(compute='_compute_work_location')
 
     ##########################################################################
     #                             FIELDS METHODS                             #
     ##########################################################################
 
+    @api.multi
+    def _compute_work_location(self):
+        for employee in self:
+            actual_location = self.env['hr.attendance'].search([
+                ('employee_id', '=', employee.id)])
```

Comment:
Here you will loop over all attendances of the employee which is very inefficient. You could instead only take his last attendance by doing 
```
actual_location = self.env['hr.attendance'].search([
    ('employee_id', '=', employee.id),
    ('check_out', '=', False)], limit=1)
```
It works because the attendances are by default sorted from the latest to the oldest.